docs/source/developer/georchestra_integration/index.rst

```diff
@@ -77,14 +78,14 @@ Setup Database
 2. Create schemas and tables as per `GeoStore documentation <https://github.com/geosolutions-it/geostore/tree/master/doc>`_
 
 Recreate MapStore Container to pick up the change
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------
 
  .. code-block:: console
 
     docker-compose up -d --force-recreate mapstore
 
 Update “header” application index to link to Mapstore
```

Comment:
This part can be removed since related changes have been merged [here](https://github.com/georchestra/georchestra/pull/3058/files)

Reply:
ok, what about create schema and tables? Should I remove the whole file or only this part ? 
 @tdipisa, @lpasquali

Reply:
We can include this in #275